qgrid/grid.py

```diff
@@ -214,16 +216,26 @@ class QGridWidget(widgets.DOMWidget):
     df = Instance(pd.DataFrame)
     precision = Integer(6)
     grid_options = Unicode('', sync=True)
-    remote_js = Bool(True)
+    remote_js = Bool(False)
+
+    def __init__(self, *args, **kwargs):
+        """Initialize all variables before building the table."""
+        kwargs.setdefault('grid_options', json.dumps(defaults.grid_options))
```

Comment:
`traitlets` provides functionality for providing dynamic defaults to traits.  We should probably use that for this functionality.  The only tricky piece might be compat between IPython versions.